core/src/main/java/com/novoda/accessibility/ActionMenuItem.java

```diff
@@ -0,0 +1,273 @@
+package com.novoda.accessibility;
+
+import android.content.Intent;
+import android.content.res.Resources;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringRes;
+import android.util.Log;
+import android.view.ActionProvider;
+import android.view.ContextMenu;
+import android.view.MenuItem;
+import android.view.SubMenu;
+import android.view.View;
+
+class ActionMenuItem implements MenuItem {
+
+    private final Resources resources;
+    private final int id;
+
+    private CharSequence title;
+    private boolean enabled;
+    @Nullable
+    private OnMenuItemClickListener menuItemClickListener;
+
+    ActionMenuItem(Resources resources, int id, CharSequence title) {
+        this.resources = resources;
+        this.id = id;
+        this.title = title;
+    }
+
+    @Override
+    public int getItemId() {
+        return id;
+    }
+
+    @Override
+    public int getGroupId() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public int getOrder() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setTitle(CharSequence title) {
+        this.title = title;
+        return this;
+    }
+
+    @Override
+    public ActionMenuItem setTitle(@StringRes int title) {
+        setTitle(resources.getString(title));
+        return this;
+    }
+
+    @Override
+    public CharSequence getTitle() {
+        return title;
+    }
+
+    @Override
+    public ActionMenuItem setTitleCondensed(@Nullable CharSequence title) {
+        if (title != null) {
+            throw new UnsupportedOperationException();
+        }
+        return this;
+    }
+
+    @Override
+    public CharSequence getTitleCondensed() {
+        return title;
+    }
+
+    @Override
+    public ActionMenuItem setIcon(Drawable icon) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setIcon(int iconRes) {
+        if (iconRes != 0) {
+            throw new UnsupportedOperationException();
+        }
+        return this;
+    }
+
+    @Override
+    public Drawable getIcon() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setIntent(Intent intent) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public Intent getIntent() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setShortcut(char numericChar, char alphaChar) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setNumericShortcut(char numericChar) {
+        Log.w("!!!", "setNumericShortcut(char) is not supported but is called by MenuInflater");
```

Comment:
`MenuInflater` calls this and the Alphabetic one with some rando character - this is a no-op, but because we're not throwing the exception, I put a warning log to highlight it in some way to consumers.

Reply:
👍 I would change the tag tho'